conda_build/convert.py

```diff
@@ -201,14 +201,21 @@ def get_pure_py_file_map(t, platform):
     if len(pythons) > 1:
         raise RuntimeError("Found more than one Python dependency in package %s"
             % t.name)
-    if len(pythons) == 0:
+    elif len(pythons) == 0:
         # not a Python package
         mapping = []
-    else:
+    elif pythons[0].group(1):
         pyver = pythons[0].group(1)
 
         mapping = [(re.compile(i[0].format(pyver=pyver)),
             i[1].format(pyver=pyver)) for i in mapping]
+    else:
+        # No python version dependency was specified
+        # Only a problem when converting from windows to unix, since 
+        # the python version is part of the folder structure on unix. 
+        if source_type == 'win' and dest_type == 'unix':
```

Comment:
Really if `dest_type == 'unix'` and `source_type` is anything. As I said, if a Unix conda package depends on more than one major Python version and has files in lib/pythonX.Y it is probably incorrect (unless the package author really wanted the stuff to only work for the one Python version).